homework/index.js

```diff
@@ -30,17 +32,94 @@
     return elem;
   }
 
+  const root = document.getElementById('root');
```

Comment:
Hi Ramazan, ideally, we should put all this dangling logic(line 35 - 57) inside a function like main and then if another function like `renderRepo` or `renderContributors` require any of these variables, they should be passed as arguments to it. For example,

`listContributor` is a variable which is being passed from `main` to `renderRepo` function. Thus, the declaration of that variable on line 49 can be moved inside the main function along with other declarations which it needs.

Reply:
I tried it but after that the code did not work in a good way.